doc/atcommands.md

```diff
@@ -1,17 +1,19 @@
-//===== Hercules Documentation ===============================
-//= Atcommand List
-//===== By: ==================================================
-//= Hercules Dev Team
-//===== Current Version: =====================================
-//= 20130115
-//===== Description: =========================================
-//= List of available atcommands and their functions.
-//============================================================
+# Hercules Documentation
+## Atcommand List
+### By
+## Hercules Dev Team
+### Current Version:
+#### 20130115
+### Description:
+#### List of available atcommands and their functions.
+******
 
 This file describes the usage of in-game commands and assumes that
-'conf/atcommand.conf' has the following:
+`conf/atcommand.conf` has the following:
+```
 	atcommand_symbol : "@"
 	charcommand_symbol: "#"
+```
 
 To search for a command, write "@" before the command name (ex. @commands).
 The format of this file is as follows:
```

Comment:
the below bullets are in single line, it should be multi-lined

Reply:
fixed.